qiskit_nature/algorithms/ground_state_solvers/adapt_vqe.py

```diff
@@ -47,6 +48,7 @@ def __init__(
         threshold: float = 1e-5,
         delta: float = 1,
```

Comment:
The argument `delta` is now no longer in use. Thus, we should deprecate it. You can do that by adding the following above the `def __init__` above:
```python
@deprecate_arguments("0.4.0", {"delta": "gradient"})
```
This will redirect users of the `delta` argument to the new `gradient` argument. We can think about adding some more information later.
You will also need to add this import at the top of the file:
```python
from qiskit_nature.deprecation import deprecate_arguments
```